### ISSUE TITLE
Large memory needs to run CRC container images in basic developer experience

### ISSUE THREAD

**Issue by eschabell** [code-ready/crc#1287]
### General information

  * OS: macOS
  * Hypervisor: hyperkit
  * Did you run `crc setup` before starting it (Yes)?

## CRC version
```bash
# Put the output of `crc version`

crc version: 1.11.0+883ca49
OpenShift version: 4.4.5 (embedded in binary)
```
  
## CRC status
```bash
# Put the output of `crc status`

CRC VM:          Running
OpenShift:       Running (v4.4.5)
Disk Usage:      16.48GB of 32.72GB (Inside the CRC VM)
Cache Usage:     12.12GB
Cache Directory: /Users/eschabel/.crc/cache
```

## CRC config
```bash
# Put the output of `crc config view`

- cpus                                  : 4
- memory                                : 12288
- pull-secret-file                      : /Users/eschabel/bin/pull-secret.txt
```

## Host Operating System
```bash
# put the output of `sw_vers` in case of Mac
ProductName:	Mac OS X
ProductVersion:	10.15.4
BuildVersion:	19E287
```

### Steps to reproduce

  1.  install crc with above config
  2. install RHDM 7.5 authoring image from container catalog (see https://gitlab.com/redhatdemocentral/rhcs-retail-webshop-demo/-/blob/master/init.sh  for steps used)
  3. install EAP 7.5 basic s2i image from container catalog (see https://gitlab.com/redhatdemocentral/rhcs-retail-webshop-demo/-/blob/master/init.sh  for steps used)
  4. watch them spin up the pods and depending on order, one of the three pods will fail with "0/1 nodes are available: 1 Insufficient memory."

### Expected
That all pods spin up and deploy.... why does this developer experience CRC based on OCP 4.4 need 12 GB and the basic product images can't be used for a web front-end app on EAP and RHDM decision service backend?

This project was used succesfully on OCP 3.x with 3-6GB of memory. 

### Actual
Something should be done to provide a developer experience on their local machine that can run with a lot less memory.

### Logs
Out of memory errors kills attempts to get you logs. See attached screenshots.

![Screenshot 2020-06-05 at 10 12 46](https://user-images.githubusercontent.com/437001/83853675-e963b900-a715-11ea-8edc-520befaf94ea.png)
![Screenshot 2020-06-05 at 10 13 17](https://user-images.githubusercontent.com/437001/83853668-e668c880-a715-11ea-93e6-765466a00580.png)
![Screenshot 2020-06-05 at 10 01 27](https://user-images.githubusercontent.com/437001/83853686-ebc61300-a715-11ea-8462-cd18583c003b.png)

**Comment by gbraad** [comment]
Duplicate of multiple other resource related issues. Please file against OpenShift 4.x as this is not CRC specific.

**Comment by gbraad** [comment]
Duplicate of #557
